nav.go

```diff
@@ -525,6 +525,13 @@ func (nav *nav) checkDir(dir *dir) {
 		nav.startPreview()
 		dir.loading = true
 		dir.loadTime = now
+
+		// Cryfs filesystems always show as having 0 blocks and are always considered to be modified at current time
```

Comment:
So apparently the root cause of #1543 is that `cryfs` constantly changes the `mtime` of the mount directory. This has actually been reported in https://github.com/cryfs/cryfs/issues/327, but it looks like there is no update. I think it would be a good idea to include the issue link in the comment to explain this workaround.

Reply:
Added the comment